app/views/layouts/application.html.erb

```diff
@@ -1,3 +1,4 @@
+<% content_for :app_title, "#{I18n.t('department.name')} Signon" %>
 <% content_for :page_title, "#{yield(:title)} | #{t('department.name')} Signon" %>
```

Comment:
One line has `I18n.t('department.name')`, the next `t('department.name')`, should these be consistent?